packages/material-ui-unstyled/src/utils/appendStyleProps.test.ts

```diff
@@ -0,0 +1,56 @@
+import { expect } from 'chai';
+import appendStyleProps from './appendStyleProps';
+
+const styleProps = {
+  className: 'bar',
+  checked: true,
+};
+
+const CustomComponent = () => null;
```

Comment:
Let's inline these into their specific tests. They're not used equaly and make debugging a test needlessly hard. A test suite doesn't need to follow DRY principles. A single test probably should.